datastore/db/__init__.py

```diff
@@ -24,12 +32,12 @@
 #     return connection
 
 
-def connect_db():
+def connect_db(conn_str : str = NACHET_DB_URL, schema :str  = NACHET_SCHEMA):
```

Comment:
```suggestion
def connect_db(conn_str : str = FERTISCAN_DB_URL, schema :str  = FERTISCAN_SCHEMA):
```

Reply:
I think the default value should be the NACHET connection string because im already calling connect_db() a lot without any arguments in nachet backend 😅

Reply:
As @sylvanie85 said. The previous function assumed it was only for Nachet db, I've adapted it without causing issue for Nachet so they can adapt the BE of it to slowly specify their connection while giving an option for FertiScan to use this functionnality. The BE should have the secrets added soon